free-courses-pt_BR.md

```diff
@@ -16,6 +16,7 @@
 * [Ionic](#ionic)
 * [Java](#java)
 * [JavaScript](#javascript)
+* [Vue.js](#vue.js)
```

Comment:
Can you remove the dot from the URL. ie `#vuejs`.

Also please slot Vue.js in alphabetical order in the table of contents.